package-lock.json

```diff
@@ -0,0 +1,9571 @@
+{
```

Comment:
We use yarn instead of NPM. Could you remove the package-lock in favour of the yarn.lock?